barretenberg/acir_tests/bootstrap.sh

```diff
@@ -215,9 +215,6 @@ function run_benchmark {
 
 # TODO(https://github.com/AztecProtocol/barretenberg/issues/1254): More complete testing, including failure tests
 function bench {
-  # TODO(https://github.com/AztecProtocol/barretenberg/issues/1265) fix acir benchmarking
```

Comment:
Should this issue be closed? I see it doesn't have a description

Reply:
Done. it was the old acir benchmarks. they're probably not coming back at this point